packages/hoppscotch-app/helpers/strategies/AxiosStrategy.js

```diff
@@ -52,12 +52,13 @@ const axiosWithoutProxy = async (req, _store) => {
       cancelToken: (cancelSource && cancelSource.token) || "",
       responseType: "arraybuffer",
     })
-
     return res
   } catch (e) {
     if (axios.isCancel(e)) {
       // eslint-disable-next-line no-throw-literal
       throw "cancellation"
+    } else if (e.response.data) {
+      throw new Error(Buffer.from(e.response.data, "base64"))
```

Comment:
```suggestion
      throw new Error(Buffer.from(JSON.stringify(e.response.data, null, 2), "base64"))
```

This should beautify the error message.

Reply:
Just saw that there are unit tests. Will add those.